tf2onnx/handler.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""Opset registry."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import collections
+import inspect
+
+# pylint: disable=unused-argument,missing-docstring,invalid-name
+
+class tf_op:
+    _OPSETS = collections.OrderedDict()
+    _MAPPING = None
+
+    def __init__(self, name, domain="onnx", **kwargs):
```

Comment:
do we want to keep it consistent with onnx spec, use an empty string for domain onnx.
https://github.com/onnx/onnx/blob/master/onnx/common/constants.h#L12-L13

Reply:
fair, I will make it constants.ONNX_DOMAIN